pages/comparison.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div>
+    <h1>Comparison</h1>
+    <table v-if="appStore.currentComparison.scenarios">
+      <thead>
+        <tr>
+          <th>
+            SCENARIO
+          </th>
+          <th
+            v-for="(value, key) in appStore.currentComparison.scenarios[0]?.parameters"
+            :key="key"
+            :class="{ 'bg-info-subtle': axis === key }"
+          >
+            {{ key }}
+            <span v-if="key === axis">
+              (Axis)
+            </span>
+          </th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr
+          v-for="(scenario, index) in appStore.currentComparison.scenarios"
+          :key="index"
+          :class="{ 'text-primary': scenarioIsBaseline(scenario) }"
+        >
+          <td>
+            {{ scenarioAxisValue(scenario) }}
+            <span v-if="scenarioIsBaseline(scenario)">
+              (Baseline)
+            </span>
+          </td>
+          <td
+            v-for="(value, key) in scenario.parameters"
+            :key="key"
+            :class="{ 'bg-info-subtle': axis === key }"
+          >
+            {{ value }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </div>
+</template>
+
+<script setup lang="ts">
+import type { ParameterSet } from "~/types/parameterTypes";
+import type { Scenario } from "~/types/storeTypes";
+
+const appStore = useAppStore();
+appStore.clearScenario();
+
+const axis = computed(() => appStore.currentComparison.axis);
+
+const scenarioAxisValue = (scenario: Scenario) => {
+  if (scenario.parameters && axis.value) {
+    return scenario.parameters[axis.value];
+  }
+};
+
+const scenarioIsBaseline = (scenario: Scenario) => {
+  return scenarioAxisValue(scenario) === appStore.currentComparison.baseline;
+};
+
+watch(() => appStore.metadata, (newValue) => {
```

Comment:
Is this reason that this needs to be a watch on metadata, rather than just happening in `onMounted` that we need to wait for the metadata to be available, and we assume this will happen after the page component has been loaded?.. oh and it also runs immediately so it's fine if metadata is already there..

Reply:
I don't remember why I chose watch instead of onMounted, but this will all be refactored some day.